librz/core/tui/panels.c

```diff
@@ -28,8 +28,8 @@
 
 #define PANEL_CMD_SYMBOLS       "isq"
 #define PANEL_CMD_XREFS         "axl"
-#define PANEL_CMD_STACK         "px"
-#define PANEL_CMD_REGISTERS     "?== true `e cfg.debug`; ?! dr; ?? ar" // select dr or ar depending on cfg.debug
+#define PANEL_CMD_STACK         "xc"
+#define PANEL_CMD_REGISTERS     "drt:value/cols/name/role"
```

Comment:
doesn't this work in debug mode only?

Reply:
no also esil

Reply:
then why is there that hack later with "if the command starts with 'drt', replace it with 'art'"?

Reply:
because the user made me notice that drt does not output anything if not in debug/emu mode

Reply:
Then it was just ok to do as before, I think:
`%== true `e cfg.debug` ; %! dr; %% ar`

Reply:
no it is not because there is a function that cuts by ` ` (white space) and splits the command in tokens.
That function breaks some stuff. the whole panel code needs to be refactored to use the C api. this is a huge mess of commands and weird ways of swapping how commands are executed.

Reply:
Are you sure? It seems the user opening the issue has tried it and it worked. I also tried it briefly and it seems to work fine even in panels.